src/tyr/serializers.cc

```diff
@@ -157,6 +158,18 @@ void openlr(const valhalla::Api& api, int route_index, rapidjson::writer_wrapper
   writer.end_array();
 }
 
+void serializeWarnings(const valhalla::Api& api, rapidjson::writer_wrapper_t& writer){
+    valhalla::Info warning;
+    writer.start_array("warnings");
+    writer.start_object();
+    for(int i = 0; i < warning.warnings_size(); i++){
+       writer(std::to_string(i), warning.warnings(i).description());
+    }
+    writer("warning_count", std::to_string(warning.warnings_size()));
```

Comment:
the whole format can be like 
```
"warnings": [
  "warning 1",
  "warning 2"
]
```
you can ignore the codes for now.

Reply:
Refactoring the output of the warning as a bare array, the parser still assigns keys to each warning.
![image](https://user-images.githubusercontent.com/40151808/162079849-8cc4ca15-6b17-4752-b5a1-ebd603205e54.png)

Reply:
well, I actually really meant a flat array of strings. no object. as long as we're not actually using codes for warnings there's no point numbering them as an array is inherently numbered and any json parser will know the length.

Reply:
ah just kinda reading the first example. the array is right, but of course without the numbers in there;)

Reply:
:smile: i just pushed the latest commit with the numbers removed.